requirements.txt

```diff
@@ -5,3 +5,6 @@ scikit-learn>=0.21.2
 sqlalchemy>=1.3.6,<2
 pyarrow>=8.0.0
 pytest>=5.0.1
+fsspec>=2023.1.0
```

Comment:
I get a bit jittery when adding requirements for an "island" in the general package requirements. In the past, we've done this for `cytominer-database`. What do you think about this option?


https://github.com/cytomining/pycytominer/blob/master/setup.py#L26

Reply:
I like that!